### PR TITLE
Docker

### PR DESCRIPTION
**Pull request by dopeCape**

### CODE REVIEW

File: dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+
+# Use the official Node.js runtime as the base image
+FROM node:16
```
Comment (iiio2) [inline dockerfile]: we are not using node 16.

Comment (iiio2) [reply dockerfile]: file name should be capitalized.

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  nextjs:
+    build:
+      context: .
+      dockerfile: dockerfile # Use Dockerfile.dev for development
```
Comment (iiio2) [inline docker-compose.yaml]: filename should be capitalized

**Comment by iiio2** [comment]
[components/PricingComponents/LongForm/index.tsx](https://github.com/MetridashCore/Batman/pull/11/files#diff-87f9a271f2f05218402e76c91122803b4eff32ed16b55d1f294fb4d809273a24) this file should not come for change.

File: dockerfile (hunk 1 above)
Comment (dopeCape) [reply dockerfile]: are we using version 20?

Comment (iiio2) [reply dockerfile]: check .nvmrc file

**Comment by iiio2** [comment]
add docker ignore file.

File: components/PricingComponents/LongForm/index.tsx
Comment (iiio2) [inline components/PricingComponents/LongForm/index.tsx:1]: this file would not come in change

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
+
+# Use the official Node.js runtime as the base image
+FROM node:18
```
Comment (iiio2) [inline Dockerfile]: use node alpine image

File: Dockerfile [hunk 4]
```diff
@@ -1,6 +1,6 @@
 
 # Use the official Node.js runtime as the base image
-FROM node:18
+FROM node:18-alpine
```
Comment (iiio2) [inline Dockerfile]: use node:18.12-alpine

File: docker-compose.yaml [hunk 5]
```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  nextjs:
```
Comment (iiio2) [inline docker-compose.yaml]: write `metridash` as services

File: components/PricingComponents/LongForm/index.tsx
Comment (iiio2) [inline components/PricingComponents/LongForm/index.tsx:1]: this file will not come on change for this pr

Comment (dopeCape) [reply components/PricingComponents/LongForm/index.tsx:1]: acutally i was just checking something and my ide just formated it according to my prettier config, there are no changes tho.

Comment (iiio2) [reply components/PricingComponents/LongForm/index.tsx:1]: pls fix lint

**Pull request closed without merging**